ansible/roles/test/files/ptftests/sonic.py

```diff
@@ -77,8 +77,8 @@ def do_cmd(self, cmd):
             except socket.timeout:
                 self.log("Timeout when running command: {}".format(cmd))
                 return ""
-            except paramiko.SSHException:
-                # Possibly caused by https://github.com/paramiko/paramiko/issues/822
+            except (paramiko.SSHException, EOFError):
```

Comment:
Can these be caught all under `except Exception`? So that every exception has one reconnection attempt?

Reply:
Technically, there will be 5 attempts, but I think all exceptions could be caught here.

The reason for the `EOFError` is unknown.